src/software/gui/generic_widgets/draw_function_visualizer/draw_function_visualizer.h

```diff
@@ -39,4 +39,32 @@ class DrawFunctionVisualizer : public ZoomableQGraphicsView
     // it, so we don't have to
     QGraphicsScene* graphics_scene;
     QOpenGLWidget* open_gl_widget;
+
+   protected:
+    // Functions to be inherited by standalone_simulator_draw_function_visualizer
+
```

Comment:
nit: you can remove this comment, we usually don't talk about subclasses in the parent class